shm/Coordinates.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+class Coordinates {
+public:
+    Coordinates() = default;
```

Comment:
Po co tak?

Reply:
Na poprzedniej lekcji Mateusz pokazywał przykład, gdzie miał zdefiniowane początkowe koordynaty na 0 i też użył konstruktora bezargumentowego

Reply:
Ale ten konstruktor i tak nic nie robi więc po co go pisać?

Reply:
Moim zdaniem ani nas to pali, ani ziębi. Nie każda implementacja klasy musi posiadać domyślny konstruktor, ale w tym przypadku otrzymany obiekt miałby sens - koordynaty (0,0), które moglibyśmy chcieć ustawić na właściwą wartość później. Tj. zamiast takiej inicjalizacji: Coordinates coordinates(0,0);, można by napisać po prostu: Coordinates coordinates;.